src/main/scala/com/yahoo/examples/SparkInferTF.scala

```diff
@@ -0,0 +1,69 @@
+package com.yahoo.examples
+
+import com.yahoo.tensorflowonspark.{DFUtil, SimpleTypeParser, TFModel}
+import org.apache.spark.sql.types._
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.SparkSession
+import org.json4s._
+import org.json4s.native.JsonMethods
+
+object SparkInferTF {
```

Comment:
This is not an example. It will be the standard class for users, and thus deserve a permanent pkg. 
How about com.yahoo.ml.tfos.Inference?

Reply:
we could just put it in with the other classes then, i.e. com.yahoo.tensorflowonspark.Inference

Reply:
Yes, please move it to com.yahoo.tensorflowonspark pkg